document_ja/mincostflow.md

```diff
@@ -86,7 +86,7 @@ vector<pair<Cap, Cost>> graph.slope(int s, int t, Cap flow_limit);
 
 $F$を流量、$m$を追加した辺の本数として
```

Comment:
I think you should put spaces before and after dollar sign.

Reply:
thanks!